easybuild/easyconfigs/i/Intltool/Intltool-0.51.0-intel-2015b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'Intltool'
```

Comment:
@hajgato: ok, please change?

It would be nice to get this PR in, it's rather big, and we know what happens with big easyconfig PRs that stay around for too long...